src/CoreShop/Bundle/CoreBundle/Resources/translations/admin.en.yml

```diff
@@ -213,3 +213,5 @@ coreshop_solidify_variant_unit_definition_data_error_no_unit_definitions: 'Produ
 coreshop_solidify_variant_unit_definition_data_strategy_only_unit_definitions: 'Are you sure to apply defined unit definitions to each variant? This operation cannot be undone! After you have executed this command, each variant must be managed separately. Also, all depending variants will be closed afterwards, unsaved changes will be lost. Variants with custom unit definitions will be skipped.'
 coreshop_solidify_variant_unit_definition_data_strategy_only_unit_definitions_and_qpr: 'Are you sure to apply defined unit definitions AND quantity price rules to each variant? This operation cannot be undone! After you have executed this command, each variant must be managed separately. Also, all depending variants will be closed afterwards, unsaved changes will be lost. Variants with custom unit definitions and quantity price rules will be skipped.'
 coreshop_solidify_variant_unit_definition_data_succesfully_applied: 'All variants has been successfully solidified. Affected Variants: {0}.'
+coreshop_restore_inheritance: 'Restore inheritance'
+coreshop_restore_inheritance_message: 'Are you sure to delete this Store Values form this Product and let it inherit from its parent again?'
```

Comment:
```suggestion
coreshop_restore_inheritance_message: 'Are you sure to delete the Store Values form this Product and let it inherit from its parent again?'
```

Reply:
👍 , but I was about to merge this PR, you can change the translation later on crowdin.com

Reply:
Is it possible to update the English language on crowdin, too?

Reply:
yes